README.md

```diff
@@ -153,4 +153,12 @@ Once oAuth has been set up in GitHub the provided ID and Secret can be added to
 
 Only accounts given admin access will be able to edit the list.  This is done by adding the users GitHub ID to the .env file.  
 
-All other users will have read only access.
\ No newline at end of file
+All other users will have read only access.
+
+## Infrastructure
```

Comment:
It would be helpful for someone reading this README to understand what commands to run to deploy the infrastructure. You could at least mention the commands run by CD pipeline